StardewArchipelago/Locations/CodeInjections/Initializers/ModCodeInjectionInitializer.cs

```diff
@@ -0,0 +1,21 @@
+﻿using StardewArchipelago.Archipelago;
+using StardewModdingAPI;
+using StardewArchipelago.Stardew;
+using StardewArchipelago.Locations.CodeInjections.Modded;
+
+namespace StardewArchipelago.Locations.CodeInjections.Initializers
+{
+    public static class ModCodeInjectionInitializer
+    {
+        public static void Initialize(IMonitor monitor, IModHelper modHelper, ArchipelagoClient archipelago, LocationChecker locationChecker)
+        {
+            InitializeModdedContent(monitor, modHelper, archipelago, locationChecker);
+        }
+
+        private static void InitializeModdedContent(IMonitor monitor, IModHelper modHelper, ArchipelagoClient archipelago, LocationChecker locationChecker)
+        {
+            DeepWoodsModInjections.Initialize(monitor, modHelper, archipelago, locationChecker);

```

Comment:
Should we only Initialize these based on their individual mods being active? That seems like a waste otherwise, and even a potential source of errors